### PR TITLE
feat(components): Added a Shell script sample

### PR DESCRIPTION
**Pull request by Ark-kun**

### CODE REVIEW

File: components/sample/Shell_script/component.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Filter text
```
Comment (Bobgy) [inline components/sample/Shell_script/component.yaml:1]: Shall we name this grep? I think people want to know the pattern is a grep pattern

**Comment by Bobgy** [comment]
Thank you!

File: components/sample/Shell_script/component.yaml (hunk 1 above)
Comment (Ark-kun) [reply components/sample/Shell_script/component.yaml:1]: I'm not 100%.
We have 4 samples doing the filtering in Shell, Python, C# and R (need Java) and they all have same name to highlight they're doing the same thing.
Perhaps `Filter text with shell and grep`.

Comment (Ark-kun) [reply components/sample/Shell_script/component.yaml]: Renamed

**Pull request merged**